salt/metalk8s/metalk8s-ui/deployed.sls

```diff
@@ -0,0 +1,25 @@
+{% set kubeconfig = "/etc/kubernetes/admin.conf" %}
+{% set context = "kubernetes-admin@kubernetes" %}
+
+Create metalk8s-ui deployment:
+  kubernetes.deployment_present:
+    - name: metalk8s-ui
+    - namespace: kube-system
+    - kubeconfig: {{ kubeconfig }}
+    - context: {{ context }}
+    - source: salt://metalk8s/metalk8s-ui/files/metalk8s-ui_deployment.yaml
+  require:
+    - pkg: Install Python Kubernetes client
+
+Create metalk8s-ui service:
+  kubernetes.service_present:
+    - name: metalk8s-ui
+    - namespace: kube-system
+    - kubeconfig: {{ kubeconfig }}
+    - context: {{ context }}
+    - metadata:
+        labels:
+          run: metalk8s-ui
+        name: metalk8s-ui
```

Comment:
This ain't right. No `spec` and such? Where's the `NodePort` type declaration gone?